### PR TITLE
Upgrade arlas-stack to v14.2.0

### PR DESCRIPTION
**Pull request by MohamedHamouGisaia**

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -16,14 +16,15 @@ services:
...
-        image: gisaia/arlas-wui-hub:${ARLAS_HUB_VERSION:-14.1.2}
+        image: gisaia/arlas-wui-hub:${ARLAS_HUB_VERSION:-14.2.1}
         container_name: arlas-hub
         environment:
             - ARLAS_USE_AUTHENT=false
             - ARLAS_HUB_BASE_HREF=/hub
             - ARLAS_WUI_URL=http://${LOCAL_HOST-localhost}:81/wui
             - ARLAS_BUILDER_URL=http://${LOCAL_HOST-localhost}:81/builder
             - ARLAS_PERSISTENCE_URL=${ARLAS_PERSISTENCE_URL:-http://localhost:81/persist}
+            - ARLAS_PERMISSIONS_URL=${ARLAS_PERSISTENCE_URL:-http://localhost:81/permissions}
```
Comment (sebbousquet) [inline docker-compose.yaml]: `ARLAS_PERSISTENCE_URL`  => `ARLAS_PERMISSIONS_URL `

File: docker-compose.yaml [hunk 2]
```diff
@@ -33,14 +34,16 @@ services:
...
-        image: gisaia/arlas-wui-builder:${ARLAS_BUILDER_VERSION:-14.1.2}
+        image: gisaia/arlas-wui-builder:${ARLAS_BUILDER_VERSION:-14.2.1}
         container_name: arlas-builder
         environment:
             - ARLAS_BUILDER_BASE_HREF=/builder
             - ARLAS_USE_AUTHENT=false
             - ARLAS_WUI_URL=http://${LOCAL_HOST-localhost}:81/wui
             - ARLAS_SERVER_URL=${ARLAS_SERVER_URL:-http://localhost:81/server}
             - ARLAS_PERSISTENCE_URL=${ARLAS_PERSISTENCE_URL:-http://localhost:81/persist}
+            - ARLAS_PERMISSIONS_URL=${ARLAS_PERSISTENCE_URL:-http://localhost:81/permissions}
```
Comment (sebbousquet) [inline docker-compose.yaml]: ARLAS_PERSISTENCE_URL => ARLAS_PERMISSIONS_URL

File: docker-compose.yaml (hunk 1 above)
Comment (MohamedHamouGisaia) [reply docker-compose.yaml]: fixed

File: docker-compose.yaml (hunk 2 above)
Comment (MohamedHamouGisaia) [reply docker-compose.yaml]: fixed

**Review by sebbousquet — APPROVED** [review]

**Pull request merged**